src/main.cpp

```diff
@@ -1019,7 +1015,7 @@ int main ( int argc, char** argv )
                 Settings.Load ( CommandLineOptions );
 
                 // load translation
-                if ( bUseGUI && bUseTranslation )
+                if ( bUseTranslation )
```

Comment:
We might want to check how much of the CLI text is translated already.

Reply:
#3195 raised for this.